### ISSUE TITLE
Release blocking issues (1.19.0-beta.0, 1.18.3, 1.17.6, 1.16.10)

### ISSUE THREAD

**Issue by justaugustus** [kubernetes/release#1281]
There are a few issues blocking releases right now and I realized there's not a singular place tracking them, so I'm opening this up.

### Required

`kube-cross` and `k8s-cloud-builder`:

- [x] (https://github.com/kubernetes/release/pull/1258) kube-cross: Support building against multiple Go versions in parallel
- [x] (https://github.com/kubernetes/release/pull/1279) Revert k8s-cloud-builder changes in kube-cross and enable variant building
- [x] (https://github.com/kubernetes/release/pull/1280) k8s-cloud-builder: Build images on kube-cross v1.14.2-4 and v1.13.9-5
- [x] Update to `kube-cross:v1.13.9-5`:
  - [x] `master`: https://github.com/kubernetes/kubernetes/pull/90963
  - [x] `release-1.18`: https://github.com/kubernetes/kubernetes/pull/90964
  - [x] `release-1.17`: https://github.com/kubernetes/kubernetes/pull/90965
  - [x] `release-1.16`: https://github.com/kubernetes/kubernetes/pull/90966

Branch cut:

- [x] (https://github.com/kubernetes/release/pull/1286, https://github.com/kubernetes/release/pull/1288) [anago] Support cutting branches at RC1 instead of beta.0 - https://github.com/kubernetes/release/issues/1207
- [x] (https://github.com/kubernetes/release/pull/1293) [krel] gcbmgr: Add validation method for `GcbmgrOptions` 
 
Release notes:

- [x] Release notes on k/k fail to generate w/ new dependencies feature - https://github.com/kubernetes/release/issues/1295

### Nice-to-have

- [x] Base image updates:
  - [x] `go-runner` (1.19-only): https://github.com/kubernetes/kubernetes/pull/90804, https://github.com/kubernetes/kubernetes/pull/90674
  - [x] `debian-base` / `debian-iptables`:
    - [x] `master`: https://github.com/kubernetes/kubernetes/pull/90665, https://github.com/kubernetes/kubernetes/pull/90697, and https://github.com/kubernetes/kubernetes/pull/90782
    - [x] `release-1.18`: https://github.com/kubernetes/kubernetes/pull/90863
    - [x] `release-1.17`: https://github.com/kubernetes/kubernetes/pull/90938
    - [x] `release-1.16`: https://github.com/kubernetes/kubernetes/pull/90940
- [x] Mismatched or dirty local k/release copy does not block krel gcbmgr runs - https://github.com/kubernetes/release/issues/1278

/assign
/kind bug
/priority critical-urgent
cc: @kubernetes/release-engineering

**Comment by saschagrunert** [comment]
Proposal for the branch cutting changes:
1. Add a new variant `beta` to the current `krel gcbmgr --type` flag, which is currently:
   `--type string            Release type (must be one of: 'prerelease', 'rc', 'official')`
2. Pass that flag via the GCB substitutions to anago
3. Modify anago (this seems to be the funny part) to not enforce `alpha` releases on the master and allow betas by taking the new release type into account. I guess one (of some) critical code part(s) is here: https://github.com/kubernetes/release/blob/c600d12013979543fcfab24c977ffae158405a21/anago#L471-L492

**Comment by justaugustus** [comment]
> Proposal for the branch cutting changes:

@saschagrunert -- Let's bring this discussion to the branch strategy issue. Commented here: https://github.com/kubernetes/release/issues/1207#issuecomment-628412415

**Comment by justaugustus** [comment]
This is complete.
/close
